### PR TITLE
Refactor lightbox and add data protection

### LINKED ISSUE #13: Add data protection lightbox
The data protection link (shown after clicking on "Ihre Daten sind sicher") should show the data protection info in a lightbox instead of a new page, similar to the other links.

Implementation detail:

To avoid having to touch all the show/hide functions and nesting another callback for the data protection lightbox we should refactor that code using events.

### PR DESCRIPTION
**Pull request by gbirke**
Added "data protection" as lightbox, as requested by fundraising team.

Refactored show/hide code into a generic class with event listeners to
avoid nested closeXXXBox function calls.

This closes #13

### CODE REVIEW

File: sensitive-banner-static/res/lightbox.css [hunk 1]
```diff
@@ -19,6 +19,17 @@
     width: 390px;
 }
 
+.banner-lightbox-dataprotection {
```
Comment (WMDE-Fisch) [inline sensitive-banner-static/res/lightbox.css]: you could merge that with the upper one

File: sensitive-banner-static/res/lightbox.css [hunk 2]
```diff
@@ -19,6 +19,17 @@
...
+    width: 390px;
+}
+
+@media screen and (min-width: 1250px) {
```
Comment (WMDE-Fisch) [inline sensitive-banner-static/res/lightbox.css]: you could merge that with the corresponding line in the sensitive banner .css ( we could also move all lightbox related stuff to one the two )

**Comment by WMDE-Fisch** [comment]
apart minor comments +1 

great job

**Comment by wmde-manicki** [comment]
The change is very good! I'll be at least +1 after minor things pointed by @WMDE-Fisch are addressed.

I really don't want to be annoying, just little thing came to my mind when looking at this new infobox. In the narrow window the new data protection "infobox" opens above this little box with "SSL-verschlüsselte Daten" and "Ihre Daten sind sicher". Same as other "infoboxes" in the narrow window, so that is generally ok, I guess. But wouldn't it be more intuitive for this particular box to show up below both two tiny boxes?

**Comment by WMDE-Fisch** [comment]
I vote for having it between these two ;-) ( but one ... probably Leszek :-D .... could go over to Henning and ask him )

**Comment by gbirke** [comment]
I'm not sure we can do this: It's a matter of source code placement. I've tried placing the data protection box after the small box and it will lead to a very big black gap above it (the size of the upper small box). Using negative margin does not work. I'm reluctant to rewrite all the lightbox CSS with absolute/relaitive positioning instead of the floats it uses at the moment. But if Henning insists, I'll do it.

**Comment by wmde-manicki** [comment]
I have asked @snaterlicious then and he said that his first impression would be to have the data protection box opened below the box that actually caused it to open.
But I wouldn't say that @snaterlicious or anyone is insisting about the placement, though. I'd say that if it is really painful to do it then do not change this in this PR. Let's do other urgent stuff and then get back to this.

**Pull request merged**